### DISCUSSION TITLE
How to prevent the {PiP: ruamel-yaml (distutils)} and {Conda: DLL} errors when installing datapane

### DISCUSSION THREAD

**Discussion by ahull002** [datapane/datapane#80] [Help]
**System:** O365/WIN10

**IDE:** JupyterLab 3.0.7

**Language:** Python version 3.8.5

**Library:** datapane

**Issue:** 
Hello, I keep getting package errors when trying to install the datapane. The errors surround ruamel-yaml (distutils) and DLL.

**Steps taken:**

I have tried installing this two ways via PiP and Conda:

**1. Via PiP:** 

`"ERROR: Cannot uninstall 'ruamel-yaml.' It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall."`

**2. Via Conda:**
`ImportError: DLL load failed while importing lib: The specified procedure could not be found.`

Any assistance or light to help me navigate what these issues are and how to get around them would be awesome. Thank you.

**Comment by lanthias** [comment]
Thanks a lot, I've converted this to an issue https://github.com/datapane/datapane/issues/84 so we can track it there.
